charts/fluent-operator/values.yaml

```diff
@@ -22,9 +24,10 @@ operator:
       requests:
         cpu: 50m
         memory: 64Mi
-  container:
-    repository: "kubesphere/fluent-operator"
-    tag: "latest"
+  image:
+    registry: ghcr.io
+    repository: fluent/fluent-operator/fluent-operator
+    tag: ""
```

Comment:
See here we keep this empty so it will fall back to the default as given to the function. https://github.com/fluent/fluent-operator/pull/1550/files#diff-6c8d42f6efe016f3bdd41422bd264c9cd013e130a6e45b41b3f5df925046f042R90

Unless someone chooses different by overriding the value, then that override is choosen.